feg/gateway/plmn_filter/plmn_filter.go

```diff
@@ -0,0 +1,57 @@
+/*
+Copyright 2020 The Magma Authors.
+
+This source code is licensed under the BSD-style license found in the
+LICENSE file in the root directory of this source tree.
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package plmn_filter
+
+import (
+	"github.com/golang/glog"
+)
+
+type PlmnIdVals map[string]PlmnIdVal
+
+type PlmnIdVal struct {
+	l5 bool
```

Comment:
I am a little confused about these `l5`, `b6` names.. what do they mean?

Reply:
So `l5` is set if the length is 5 and `b6` is what ever the 6th character is? Not sure if this naming scheme is due to some convention, but can we either change the name or leave a comment?

Reply:
Note we create a map with a key of lenght 5. So having `00101` and `001011` uses the same key.
- `l5` is a boolean and it is used when the plmn is of lenght 5 (Thats easy)
- 'b6` If it exist, that means PLMN is 6 and b6 stores that sixth number.

so for the example you would have 
```
{ `00101`:{l5:true, b6:byte{1}} }
```

However, note this is fancy but per I could read normally the providers will not have overlapping PLMNs.

Reply:
Got it pls add comments :) I will forget in no time.